tests/test_structures.py

```diff
@@ -41,20 +45,39 @@ def test_structure_manager_widget(structure_data_object):
         structure_manager_widget.structure[0].position == new_structure[0].position
     )
 
+    # Store the modified structure.
+    structure_manager_widget.btn_store.click()
+    assert structure_manager_widget.structure_node.is_stored
+    stored_structure = structure_manager_widget.structure_node
+
     # Undo the structure modification.
     structure_manager_widget.undo()
     assert np.any(
         structure_manager_widget.structure[0].position != new_structure[0].position
     )
+    structure_manager_widget.undo()  # Undo the structure creation.
+    assert structure_manager_widget.structure is None
 
-    # test the widget can be instantiated with empty inputs
+    # Test the widget with multiple importers, editors. Specify the viewer and the node class
```

Comment:
This test is already very long, it would be good to split this up. Feel free to do in a follow up.

Reply:
done in https://github.com/aiidalab/aiidalab-widgets-base/pull/373/commits/cb6e6bb83df0f4ebe7086e8e9b69a1e4254691fc